tests/state/test_replacements_bypass_projects.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime, timedelta
+
+import pytest
+from freezegun import freeze_time
+
+from snuba.replacers.replacements_utils import (
+    EXPIRY_WINDOW,
+    get_config_auto_replacements_bypass_projects,
+    set_config_auto_replacements_bypass_projects,
+)
+
+
+@freeze_time("2024-5-13 09:00:00")
+class TestState:
+    start_test_time = datetime.now()
+    proj1_add_time = start_test_time
+    proj2_add_time = start_test_time + timedelta(minutes=2)
```

Comment:
```suggestion
    proj2_add_time = start_test_time + timedelta(minutes=EXPIRY_WINDOW_MINUTES // 2)
```

The reader of this file shouldn't have to *know* that the EXPIRY_WINDOW is 5 minutes long and that 2 is less than 5. Just one project expires sooner than the other